Resources/config/doctrine/ScheduledCommand.orm.yml

```diff
@@ -62,6 +62,12 @@ JMose\CommandSchedulerBundle\Entity\ScheduledCommand:
             unique: false
             nullable: false
             column: PRIORITY
+        executionMode:
+            type: string
+            length: 50
+            unique: false
+            nullable: false
```

Comment:
- This feature must be compatible with existing installations, in any case I setted all Commands as Auto in the _ScheduledCommand::__construct()_

```
public function __construct()
    {
        $this->setLastExecution(new \DateTime());
        $this->setLocked(false);
        $this->setExecutionMode(self::MODE_AUTO);
    }
```